views/js/qtiPrintRenderer/renderers/choices/SimpleChoice.ChoiceInteraction.js

```diff
@@ -27,6 +27,20 @@ define([
 ], function(tpl, getContainer){
     'use strict';
 
+    /**
+     * Detects image of relative size in choice
+     * @param {jQuery} $container 
+     * @returns {boolean} flag if simple choice has relative sized image
+     */
+    function hasRelativeSizedImage($container) {
+        let flag = false;
+        const $img = $container.find('img');
+        if($img && $img.length && $img.attr('width') && ~$img.attr('width').indexOf('%')) {
```

Comment:
**question:** Was it intended to use a bitwise operator? Is it a trick to consider `-1` as `false` and other values as `true`?

Reply:
Yes, it is  shorthand for `.indexOf('str') !== -1`.  Only `~-1 === 0`, so other values will be truthy.

Do you think it's better to avoid this pattern as it is a bit blurry?

Reply:
Well, personally I like the trick and I find it elegant.
However, it looks weird to readers that are not used to bitwise operations.

Some reading:
- https://www.codecademy.com/resources/docs/javascript/bitwise-operators
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_NOT